From: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Date: Wed, 25 Nov 2009 18:12:48 +0100
Subject: [PATCH] libata: add private driver field to struct ata_device
Git-commit: 95514fd8ff0f30de7815950edfd84ef1e19fb1c8
References: FATE#311054
Patch-Mainline: 2.6.33

This brings struct ata_device in-line with struct ata_{port,host}.

Signed-off-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Jeff Garzik <jgarzik@redhat.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 include/linux/libata.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/libata.h b/include/linux/libata.h
index 82e6a98..e44d14a 100644
--- a/include/linux/libata.h
+++ b/include/linux/libata.h
@@ -599,6 +599,7 @@ struct ata_device {
 	unsigned int		horkage;	/* List of broken features */
 	unsigned long		flags;		/* ATA_DFLAG_xxx */
 	struct scsi_device	*sdev;		/* attached SCSI device */
+	void			*private_data;
 #ifdef CONFIG_ATA_ACPI
 	acpi_handle		acpi_handle;
 	union acpi_object	*gtf_cache;
-- 
1.6.0.2

